src/Symfony/Bundle/FrameworkBundle/Command/DotenvDebugCommand.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Dotenv\Dotenv;
+
+/**
+ * A console command to debug current dotenv files with variables and values.
+ *
+ * @author Christopher Hertel <mail@christopher-hertel.de>
+ */
+final class DotenvDebugCommand extends Command
+{
+    protected static $defaultName = 'debug:dotenv';
+    protected static $defaultDescription = 'Lists all dotenv files with variables and values';
+
+    private $kernelEnvironment;
+    private $projectDirectory;
+
+    public function __construct(string $kernelEnvironment, string $projectDirectory)
+    {
+        $this->kernelEnvironment = $kernelEnvironment;
+        $this->projectDirectory = $projectDirectory;
+
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $io->title('Dotenv Variables & Files');
+
+        $envFiles = $this->getEnvFiles();
+        $availableFiles = array_filter($envFiles, function (string $file) {
+            return is_file($this->getFilePath($file));
+        });
+
+        if (\in_array('.env.local.php', $availableFiles, true)) {
+            $io->warning('Due to existing dump file (.env.local.php) all other dotenv files are skipped.');
+        }
+
+        if (is_file($this->getFilePath('.env')) && is_file($this->getFilePath('.env.dist'))) {
+            $io->warning('The file .env.dist gets skipped due to the existence of .env.');
+        }
+
+        $io->section('Scanned Files (in descending priority)');
+        $io->listing(array_map(static function (string $envFile) use ($availableFiles) {
+            return \in_array($envFile, $availableFiles, true)
+                ? sprintf('<fg=green>✓</> %s', $envFile)
+                : sprintf('<fg=red>⨯</> %s', $envFile);
```

Comment:
the table headers would convey the same.

i feel like this checkmark should convery if Dotenv "parsed" it for merging yes/no